test/test_network_constructors.jl

```diff
@@ -162,6 +165,46 @@ end
     end
 end
 
+@testset "Test Locational Marginal Prices between DC lossless with PowerModels vs StandardPTDFModel" begin
+    network = [DCPPowerModel, StandardPTDFModel]
+    sys = build_system("c_sys5")
+    dual_constraint = [
+        [PSI.make_constraint_name(PSI.NODAL_BALANCE_ACTIVE, PSY.Bus)],
```

Comment:
It's better not to expose internal methods here. We should add a better interface to request the duals than passing the symbols but that is another issue.

Reply:
I can replace it with just symbols

Reply:
Yes, that's better. This PR reflected some pending work with the interfaces